sbncode/SBNEventWeight/Calculators/BNBFlux/FluxCalcPrep.cxx

```diff
@@ -250,8 +251,9 @@ namespace sbn {
           throw cet::exception(__FUNCTION__) << GetName()<<"::Unknown ntype "<<fluxlist[inu].fntype<< std::endl;
         }
 
-        // Collect neutrino energy
-        double enu= mclist[inu].GetNeutrino().Nu().E();
+        // Collect neutrino energy; mclist is replaced with fluxlist.
+//        double enu= mclist[inu].GetNeutrino().Nu().E();
+        double enu= fluxlist[inu].fnenergyn;
```

Comment:
Just to verify, did you make sure that the units are consistent between `mclist[inu].GetNeutrino().Nu().E()` and `fluxlist[inu].fnenergyn`?

Reply:
Yes I got the same number from that variable.